lib/generators/bigbluebutton_rails/templates/migration_3_1_0.rb

```diff
@@ -1,6 +1,76 @@
 class BigbluebuttonRailsTo310 < ActiveRecord::Migration
-  def change
-    drop_table :bigbluebutton_server_configs
```

Comment:
What happened to these `drop_table`s?
I'm asking because you create these tables on `down`, but don't drop them  on `up`.

Reply:
Answer: they were already dropped before, on an older migration.